tests/Search20Test.php

```diff
@@ -113,23 +113,6 @@ public function testSearchByKeyboardId()
       $this->assertEquals('keyboard_id', $json->keyboards[0]->match->type);
     }
 
-    public function testSearchByLegacyKeyboardId()
```

Comment:
Does this get removed in a `Search30Test.php`?

Reply:
Sorry, I don't understand the question?

Reply:
Search20Test.php uses /search/2.0/search.json
https://github.com/keymanapp/api.keyman.com/blob/be87e96ec10df73534eb758403e1fdcd88f8d55c/tests/Search20Test.php#L14C37-L14C61

Do we need a Search30Test.php that uses /search/3.0/search.json for tests?

Reply:
No, because this is a test of the search 2.0 api, and we haven't got a search 3.0 api.

Reply:
Oh wait

Reply:
We've got a search 3.0 schema but not a search 3.0 api. Confuse much? Basically search 3.0 less data than search 2.0.